src/renderer/components/multimachinebackups/SelectSiteBackup.scss

```diff
@@ -1,3 +1,5 @@
+@import '~@getflywheel/local-components/src/styles/_partials/index';
```

Comment:
Curious what the tilde is doing here?

Reply:
tilde means, assume we're starting at `node_modules`.